docs/03-endpoints/api-v2/knora-iris.md

```diff
@@ -209,7 +211,7 @@ follows:
     `http://rdfh.ch/projects/PROJECT_SHORTCODE/mappings/MAPPING_NAME`
   - XML-to-standoff mapping element:
     `http://rdfh.ch/projects/PROJECT_SHORTCODE/mappings/MAPPING_NAME/elements/MAPPING_ELEMENT_UUID`
-  - Project: `http://rdfh.ch/projects/PROJECT_UUID`
+  - Project: `http://rdfh.ch/projects/PROJECT_SHORTCODE` (or `http://rdfh.ch/projects/PROJECT_UUID`)
```

Comment:
The "(or `http://rdfh.ch/projects/PROJECT_UUID`)" is only valid for the BEOL project, right? (Nothing wrong with it, I just want to make sure that I understood it correctly.)

Reply:
Correct, after revert is't only valid for BEOL, however it also makes open the way to go for new projects having UUID in the IRI instead of the shortcode.